helpers_objectstorage.go

```diff
@@ -110,6 +110,11 @@ var preauthenticatedRequestSchema = map[string]*schema.Schema{
 			string(baremetal.PARObjectReadWrite),
 			string(baremetal.ObjectRead)}, true),
 	},
+	"access_uri": {
```

Comment:
Access_type is required for create according to the spec 
https://bitbucket.aka.lgl.grungy.us/projects/CASPER/repos/casper-api-spec/browse/src/main/resources/casper-api.yaml